safe-global/safe-wallet-web
# fix: Remove WC onboarding tooltip

## What it solves

As discussed on [Slack](https://5afe.slack.com/archives/C04MQSC3R7Y/p1699123662614129?thread_ts=1699115887.638169&cid=C04MQSC3R7Y) we are removing the WC Onboarding tooltip.

## How this PR fixes it

Removes the WC Onboarding Tooltip

Also fixes the Onboard Safe logo size on mobile

## How to test it

1. Open a Safe
2. Observe no WC tooltip in the header

## Screenshots
<img width="367" alt="Screenshot 2023-11-06 at 16 39 28" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/118b501d-5bca-46eb-9780-08394c882faf">

## Checklist
